CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ meeting](https://docs.google.com/document/d/1l_M0om0qUEN_NNiGgpqJ2tvsF2iioHkaARD
 ### How to run the test suite
 
 Prerequisites:
-* go >= 1.15
+* go >= 1.16
 * kubebuilder >= 2.3
```

Comment:
I think the controller-runtime v0.9.0 release was combined with the release of kubebuilder v3.x.x